docs/src/extensions.md

```diff
@@ -1,4 +1,4 @@
-# Extensions over GNU
+: Extensions over GNU
```

Comment:
: ?

Reply:
I ... don't know what happened there, that's a mistake

Reply:
Thanks for catching it!